compiler/rustc_trait_selection/src/traits/coherence.rs

```diff
@@ -425,7 +425,8 @@ fn plug_infer_with_placeholders<'tcx>(
             if ty.is_ty_var() {
                 let Ok(InferOk { value: (), obligations }) =
                     self.infcx.at(&ObligationCause::dummy(), ty::ParamEnv::empty()).eq(
-                        DefineOpaqueTypes::No,
+                        // Already using new solver, so `DefineOpaqueTypes` is unused
```

Comment:
Justification for these three are not exactly correct -- we are always unifying with an infer var, so we never expect to see an opaque anyways.